### PR TITLE
IfcDistributionBoard Rail Import BIMQ 2021-07-21T07:13:47.0000262Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcDistributionBoard

### CODE REVIEW

File: IFC4x3/Properties/p/PortCapacity_2wGIZukqb5y8_rmxPBWB8h/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Indicates the cross port capacity of the distribution frame.
```
Comment (larswik) [inline IFC4x3/Properties/p/PortCapacity_2wGIZukqb5y8_rmxPBWB8h/Documentation.md:1]: Is this completely clear (PortCapacity : IfcInteger)? Excuse my ignorance.

File: IFC4x3/Properties/i/IK_Code_2kpG9_rK93LwNz4pCQcRnH/DocProperty.xml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IK_Code_2kpG9_rK93LwNz4pCQcRnH" Name="IK_Code" UniqueId="aecd027e-d542-4357-a5fd-13331a99bc51" />
```
[the remark this answers is not in the document]
Comment (larswik) [reply IFC4x3/Properties/i/IK_Code_2kpG9_rK93LwNz4pCQcRnH/DocProperty.xml:2]: Just checking if this is handled now. IK_Code seems to be a property from Pset_ElectricalDeviceCommon.

File: IFC4x3/Properties/p/PortCapacity_2wGIZukqb5y8_rmxPBWB8h/Documentation.md (hunk 1 above)
Comment (czapplitec) [reply IFC4x3/Properties/p/PortCapacity_2wGIZukqb5y8_rmxPBWB8h/Documentation.md]: Updated the documentation: https://github.com/bSI-InfraRoom/IFC-Specification/pull/173/commits/4ed676f1d10748cf1dc4a55f7049ef2ff53de67b

**Review by larswik — APPROVED** [review]

File: IFC4x3/Properties/p/PortCapacity_2wGIZukqb5y8_rmxPBWB8h/DocProperty.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PortCapacity_2wGIZukqb5y8_rmxPBWB8h" Name="PortCapacity" UniqueId="ba4128f8-bb49-45f0-8fb5-c3b64b80b22b" PrimaryDataType="IfcInteger" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/PortCapacity_2wGIZukqb5y8_rmxPBWB8h/DocProperty.xml:2]: Is this not expected to be modeled?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DistributionBoardTypeDispatchingBoard/Documentation.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Properties for _IfcDistributionBoard_ with PredefinedType _DISPATCHINGBOARD_.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DistributionBoardTypeDispatchingBoard/Documentation.md]:
```suggestion
Properties for _IfcDistributionBoard_ with PredefinedType DISPATCHINGBOARD.
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DistributionBoardTypeDistributionFrame/Documentation.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Properties for _IfcDistributionBoard_ with PredefinedType _DISTRIBUTIONFRAME_.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DistributionBoardTypeDistributionFrame/Documentation.md]:
```suggestion
Properties for _IfcDistributionBoard_ with PredefinedType DISTRIBUTIONFRAME.
```

**Review by SergejMuhic — COMMENTED** [review]
@christophe-blt and @larswik are you guys happy with this one? I cannot judge whether your comments have been address properly.

I also proposed a change to formatting of the predefined type. If I am not mistaken, it was handled as I have proposed in other psets.

File: IFC4x3/Properties/n/NumberOfInterfaces_0x8qDQiJjFd9ZlduDoHwXH/DocProperty.xml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfInterfaces_0x8qDQiJjFd9ZlduDoHwXH" Name="NumberOfInterfaces" UniqueId="3b23435a-b13b-4f9c-98ef-9f837247a851" PropertyType="p_tablevalue" PrimaryDataType="IfcLabel" />
```
[the remark this answers is not in the document]
Comment (czapplitec) [reply IFC4x3/Properties/n/NumberOfInterfaces_0x8qDQiJjFd9ZlduDoHwXH/DocProperty.xml]: This one is reused: https://github.com/bSI-InfraRoom/IFC-Specification/pull/173/commits/33f443ccebebcf43b1e6bf50d0b7d57bc24ffdc9

File: IFC4x3/Properties/p/PortCapacity_2wGIZukqb5y8_rmxPBWB8h/DocProperty.xml (hunk 3 above)
Comment (czapplitec) [reply IFC4x3/Properties/p/PortCapacity_2wGIZukqb5y8_rmxPBWB8h/DocProperty.xml:2]: This is not expected to be modeled by IfcDistributionPort. This only states port connections that can be supported by the distribution frame. It is fixed information for the product.

File: IFC4x3/Properties/p/PortCapacity_2wGIZukqb5y8_rmxPBWB8h/Documentation.md (hunk 1 above)
Comment (SergejMuhic) [reply IFC4x3/Properties/p/PortCapacity_2wGIZukqb5y8_rmxPBWB8h/Documentation.md]: @larswik is this resolved in your view?

**Review by AlexBrad1eyCT — APPROVED** [review]
This seems all Good LFTM

**Review by AlexBrad1eyCT — APPROVED** [review]

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**